esmvaltool/diag_scripts/ocean/diagnostic_maps_multimodel.py

```diff
@@ -0,0 +1,316 @@
+"""Model vs Observations maps Diagnostic.
+
+Diagnostic to produce comparison maps of model(s) and data (if provided).
+If observations are not provided, data maps for each model are drawn.
+
+The image shows on top row observational data and the following subplot(s)
+the comparison for each model by following the order of the recipe.
+
+Note that this diagnostic assumes that the preprocessors do the bulk of the
+hard work, and that the cube received by this diagnostic (via the settings.yml
+and metadata.yml files) has no time component, a small number of depth layers,
+and a latitude and longitude coordinates.
+
+An approproate preprocessor for a 2D + time field would be::
+
+  preprocessors:
+    prep_map:
+      time_average:
+      regrid:
+        target_grid: 1x1
+        scheme: linear
+
+Author: lova_to
+"""
+import logging
+import os
+import sys
+
+import matplotlib.pyplot as plt
```

Comment:
Please sort your imports according to pep8, use `isort` to do it automatically

Reply:
thanks ! That was useful!